modules/create-aws-cluster.adoc

```diff
@@ -73,6 +71,15 @@ CIDR configurations cannot be changed later. Confirm your selections with your n
 If the cluster privacy is set to *Private*, you will not be able to access your cluster until you configure private connections in your cloud provider.
 ====
 
+. Optional: Select *Enable etcd encryption* if you require etcd key value encryption. This is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
```

Comment:
@ctauchen thank you for your detailed review and for this comment!

The etcd key values are encrypted with this option, but not the keys themselves. I have subsequently made this clearer in each instance by specifically stating that. I will leave the "key value encryption" references without the hyphen, because the key-value pairs are not being referred to here.